django/contrib/admin/actions.py

```diff
@@ -4,12 +4,14 @@
 
 from django.contrib import messages
 from django.contrib.admin import helpers
+from django.contrib.admin.decorators import action_method
 from django.contrib.admin.utils import model_ngettext
 from django.core.exceptions import PermissionDenied
 from django.template.response import TemplateResponse
 from django.utils.translation import gettext as _, gettext_lazy
 
 
+@action_method(permissions=['delete'], description=gettext_lazy("Delete selected %(verbose_name_plural)s"))
```

Comment:
Line break for future black compat? 😇

```suggestion
@action_method(
    permissions=['delete'],
    description=gettext_lazy("Delete selected %(verbose_name_plural)s"),
)
```

Reply:
Black won't mind. 😀

Reply:
I guess the description is quite long, so probably worth breaking the line.